script.js

```diff
@@ -0,0 +1,18 @@
+const openMobile = document.getElementById('bars');
+const closeMobile = document.getElementById('close');
+const mobilemenu = document.getElementById('mobile_m');
+const menuList = document.getElementsByClassName('navlist');
+console.log(menuList);
```

Comment:
- [ Required ] I think it would be better if you remove this `console.log()` statement from this file. It's not good practice to have `console.log` statements in your production code, console.log statements if left in projects can at times log out sensitive information such as passwords.